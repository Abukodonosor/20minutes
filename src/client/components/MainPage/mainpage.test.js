import React from "react";
import renderer from "react-test-renderer";

import { MainPage, BlogItemCard } from "./index.jsx";

describe("MainPage component", () => {
  it("snapshot renders", () => {
    const component = renderer.create(<MainPage />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
