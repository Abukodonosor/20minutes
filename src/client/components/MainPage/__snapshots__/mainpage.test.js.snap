// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage component snapshot renders 1`] = `
<div
  className="main-page"
/>
`;
